### PR TITLE
Updates for compatibility with R-4.0 and dplyr 1.0.0

### PR DESCRIPTION
**Pull request by ytakemon**
In progress: updates to the dplyr lesson. 

Please Do not merge until updates are completed and reviewed at the monthly maintainers meeting.

### CODE REVIEW

**Comment by ytakemon** [comment]
@kpoterlowicz Ready for your review!

**Comment by kpoterlowicz** [comment]
@ytakemon should I be concern about build website fail

File: _episodes_rmd/04-dplyr.Rmd [hunk 1]
```diff
@@ -70,7 +73,7 @@ every time you open a new R session and want to use that package.
 ## What is dplyr?
 
 The package `dplyr` is a fairly new (2014) package that tries to provide easy
-tools for the most common data manipulation tasks. It is built to work directly
+tools for the most common data manipulation tasks. This package is also included larger collection of the `tidyverse` package. It is built to work directly
```
Comment (kpoterlowicz) [inline _episodes_rmd/04-dplyr.Rmd:76]: please check the sentence

File: _episodes_rmd/04-dplyr.Rmd [hunk 2]
```diff
@@ -85,11 +88,19 @@ memory. The database connections essentially remove that limitation in that you
 can have a database of many 100s GB, conduct queries on it directly and pull
```
Comment (kpoterlowicz) [inline _episodes_rmd/04-dplyr.Rmd:88]: rephrase 'many 100s GB'

Comment (kpoterlowicz) [inline _episodes_rmd/04-dplyr.Rmd:88]: rephrase 'many 100s GB'

File: _episodes_rmd/04-dplyr.Rmd [hunk 3]
```diff
@@ -210,21 +231,21 @@ to a probability value according to the formula:
...
 
 > ## Exercise
 > There are a lot of columns in our dataset, so let's just look at the
-> `sample_id`, `POS`, `QUAL`, and `POLPROB` columns for now. Add a 
-> line to the above code to only show those columns. 
-> 
+> `sample_id`, `POS`, `QUAL`, and `POLPROB` columns for now. Add a
+> line to the above code to only show those columns.
+>
 >> ## Solution
```
Comment (kpoterlowicz) [inline _episodes_rmd/04-dplyr.Rmd:249]: quite a long output

File: _episodes_rmd/04-dplyr.Rmd [hunk 4]
```diff
@@ -290,4 +333,4 @@ variants %>%
```
Comment (kpoterlowicz) [inline _episodes_rmd/04-dplyr.Rmd:333]: shall we have an example of us  gather and spread?

**Review by kpoterlowicz — APPROVED** [review]
please see minor comments

**Comment by JasonJWilliamsNY** [comment]
> @ytakemon should I be concern about build website fail

I don't think so - I think we are all failing, but when the content is about ready we should reconnect with @fmichonneau or @ErinBecker who can advise on format and styling. Getting there!

**Comment by fmichonneau** [comment]
this particular failure is safe to ignore. I'll open up a PR with the latest version of the template that takes care of this particular issue related to remote themes.

File: _episodes_rmd/04-dplyr.Rmd (hunk 3 above)
Comment (ytakemon) [reply _episodes_rmd/04-dplyr.Rmd:249]: new edits were committed addressing this.

**Comment by ytakemon** [comment]
@kpoterlowicz I've added edits to address some of your comments. Please take another look and approve if everything looks fine.

**Pull request merged**